shuup/admin/modules/orders/toolbar.py

```diff
@@ -72,50 +76,59 @@ def _build_order_set_state_button(self):
             )
 
     def _build_set_complete_button(self):
-        self.append(PostActionButton(
-            post_url=reverse("shuup_admin:order.set-status", kwargs={"pk": self.order.pk}),
-            name="status",
-            value=OrderStatus.objects.get_default_complete().pk,
-            text=_("Set Complete"),
-            icon="fa fa-check-circle",
-            disable_reason=(
-                _("This order can not be set as complete at this point")
-                if not self.order.can_set_complete()
-                else None
-            ),
-            extra_css_class="btn-success"
-        ))
+        self.append(
+            PostActionButton(
+                post_url=reverse("shuup_admin:order.set-status", kwargs={"pk": self.order.pk}),
+                name="status",
+                value=OrderStatus.objects.get_default_complete().pk,
+                text=_("Set Complete"),
+                icon="fa fa-check-circle",
+                disable_reason=(
+                    _("This order can not be set as complete at this point")
+                    if not self.order.can_set_complete()
+                    else None
+                ),
+                extra_css_class="btn-success",
+            )
+        )
 
     def _build_cancel_button(self):
-        self.append(PostActionButton(
-            post_url=reverse("shuup_admin:order.set-status", kwargs={"pk": self.order.pk}),
-            name="status",
-            value=OrderStatus.objects.get_default_canceled().pk,
-            text=_("Cancel Order"),
-            icon="fa fa-trash",
-            disable_reason=(
-                _("Paid, shipped, or canceled orders cannot be canceled")
-                if not self.order.can_set_canceled()
-                else None
-            ),
-            extra_css_class="btn-danger btn-inverse"
-        ))
+        self.append(
+            PostActionButton(
```

Comment:
same as above, this status will be available in `_build_order_set_state_button`, right?